conf/local.conf.js

```diff
@@ -1,8 +1,9 @@
-var browserstack = require('browserstack-local');
 require('dotenv').config();
+var browserstack = require('browserstack-local');
 
 const user = process.env.BROWSERSTACK_USERNAME || '';
 const key = process.env.BROWSERSTACK_ACCESS_KEY || '';
+const isProduction = process.env.NODE_ENV === 'production';
```

Comment:
Capabilities question: I see we play between iPhone 8 and iPhone 12. Is there an iPhone which is available for XCode and for Browserstack Cloud?

```
platformName: "iOS",
automationName: "XCUITest",
deviceName: "iPhone 8 Plus",
platformVersion: "14.4",
browserName: "Safari",
```

Reply:
I guess we will have to have 2 configurations. One for local test development using appium, and another for the playground pipeline that goes to browserstack. Each having different devices